commcare_connect/opportunity/views.py

```diff
@@ -1222,6 +1208,28 @@ def payment_report(request, org_slug, pk):
     )
 
 
+class PaymentInvoiceTableView(OrganizationUserMixin, SingleTableView):
```

Comment:
Where is this view getting used? I don't see it in urls.py

Reply:
This is not a part of this PR. I am going to remove this.